bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -232,27 +242,32 @@ protected static ScheduledExecutorService createDefaultRetryExecutor() {
   private final ScheduledExecutorService scheduledRetries;
   private final List<Closeable> clientCloseHandlers = Collections
       .synchronizedList(new ArrayList<Closeable>());
-  private final ClientInterceptor clientAuthInterceptor;
+  private ClientInterceptor clientAuthInterceptor;
+
+  public BigtableSession(BigtableOptions options) throws IOException {
+    this (options, null);
```

Comment:
I personally prefer to call one overloaded constructor.  example:

public BigtableSession(BigtableOptions options) throws IOException {
  this(options, null, null, null);
}